src/script_error.rs

```diff
@@ -1,14 +1,17 @@
+use std::num::TryFromIntError;
+
 use thiserror::Error;
 
 use crate::{
-    script::{Bad, Disabled},
+    interpreter,
+    script::{self, Bad, Disabled},
     signature,
 };
 
-#[derive(Copy, Clone, PartialEq, Eq, Debug, Error)]
+#[derive(Clone, PartialEq, Eq, Debug, Error)]
 pub enum ScriptNumError {
```

Comment:
If this enum was in the public API in a previous release, document all of these changes to it in the changelog.